### ISSUE TITLE
Visually "filter" inferred axioms

### ISSUE THREAD

**Issue by csnyulas** [protegeproject/swrltab-plugin#26]
In the "Inferred Axioms" tab of SWRLTab, show trivial axioms (such trivial declarations and equivalent classes axioms, subclassof axioms involving owl:Thing and owl:Nothing, etc) in grey, or otherwise de-emphasize them, so that the user can focus on the meaningful inferences.

As it can be seen in this screen shot most of the inferred axioms are uninteresting to the user:

![screenshot swrltab inferred axioms](https://cloud.githubusercontent.com/assets/2441046/13622314/b325be1c-e552-11e5-8dde-4e8afff653ef.png)

This is somewhat related to the current solution provided for #17.

**Comment by matthewhorridge** [comment]
Declarations shouldn't be inferred as they are non-logical axioms.

As a side note (and in relation with all the other issues to do with inferences), it's my understanding that once @martinjoconnor has rewritten the reasoner as a true Protege reasoner plugin, all of these issues will be irrelevant and presentation of inferred information in the Protege UI will come "for free" in the standard way.

**Comment by martinjoconnor** [comment]
Yup. These are stopgap measure to make the current functionality usable.

**Comment by csnyulas** [comment]
I totally agree with both of you. However, the two list that I am referring at in this issue, are part of the SWRLTab plugin, and I think that de-emphasizing trivial inferences in these lists would definitely help users who would like to review the inferred axioms in order to find the interesting axioms that may be relevant for them.
